### ISSUE TITLE
Will the vaccination certificate be included in my backup?

### ISSUE THREAD

**Issue by ndegendogo** [corona-warn-app/cwa-documentation#621]
## Your Question

* Documentation File:
* Line / Paragraph:
* Question:

Will the digital vaccination certificate be included in my backup?
Will it be possible to transfer it to another device, e.g. if I buy a new device?

**Comment by Ein-Tim** [comment]
Related issues:

- https://github.com/corona-warn-app/cwa-wishlist/issues/500
- https://github.com/corona-warn-app/cwa-wishlist/issues/522

**Comment by jucktnich** [comment]
The problem is, that the DGC Team doesn't want a possibility for multiple instances of the passes, right?

**Comment by thomasaugsten** [comment]
Backup is not necessary. You can scan your certificate as often you want in all different apps you want.

**Comment by jucktnich** [comment]
Ah, FAQ was updated. Just ignore my comment above

**Comment by ndegendogo** [comment]
> Backup is not necessary. You can scan your certificate as often you want in all different apps you want.

@thomasaugsten **this is really good news.**

From the [FAQ for CoVPass on the page of BMG](https://www.bundesgesundheitsministerium.de/coronavirus/faq-covid-19-impfung/faq-digitaler-impfnachweis.html) (version as of 14 May 2021) I got the impression that they plan to prevent scanning of a certificate with multiple devices, so I was worried.

But now I see that they removed these clauses (restriction to a single device) in their current version if the FAQ.

Thanks - question answered, I am happy again, and looking forward to this new planned function of cwa.

Closing this ticket.
